compiler/include/symbol.h

```diff
@@ -721,6 +726,7 @@ extern StringChainHash uniqueStringHash;
 extern Symbol *gNil;
 extern Symbol *gUnknown;
 extern Symbol *gMethodToken;
+extern Symbol *gRetByRefToken;
```

Comment:
It's kindof weird, so we definately need a comment about what it's for. (Allow calls to functions that return through a reference argument to exist without creating extra temporaries. This is needed especially for ForallStmts). But I really would like a different name, since this variable is not used normally with ReturnByRef functions the way MethodToken is used for methods. Let's call it `gDummyReturnValue`.

Reply:
Or maybe `gDummyValue`.

Reply:
I call it `gDummyRef` because it is a reference.